js/index.js

```diff
@@ -1,7 +1,44 @@
 // Iteration 1: Names and Input
 
+let hacker1 = ("Jan");
+console.log(`The driver's name is ${hacker1}`);
+
+let hacker2 = ("Bert");
+console.log(`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
 
+if (hacker1.length == hacker2.length) {
+    console.log(`Wow, both have equally long names, ${hacker1.length} characters!`);
+  } else if (hacker1.length >= hacker2.length) {
```

Comment:
Do you really want to check whether hacker1.length is longer _or equal_ to hacker2.length?